src/main/plugin/iso19139.ca.HNAP/templates/hnap_nonspatial-english.xml

```diff
@@ -157,6 +159,22 @@
          </gmd:characterEncoding>
       </gmd:PT_Locale>
   </gmd:locale>
+   <gmd:locale>
```

Comment:
Why are you adding this?  I thought this was causing issues because it is the main language?  We used to have it in the past and I thought we removed it because it was causing issues - I don't recall the issue exactly - Maybe @josegar74 remembers?

Reply:
This was the only document that did not already have both ...

Still if we wish to remove the gmd:locale matching the main language I will do so as a distinct commit